taccsite_cms/settings.py

```diff
@@ -323,6 +323,15 @@ def gettext(s): return s
 if os.path.exists(ui_demo_dir):
     STATICFILES_DIRS += (('ui', ui_demo_dir),)
 
+STORAGES = {
+    "default": {
+        "BACKEND": "django.core.files.storage.FileSystemStorage",
+    },
+    "staticfiles": {
+        "BACKEND": "django.contrib.staticfiles.storage.ManifestStaticFilesStorage",
+    },
+}
```

Comment:
**Introduction:**
We now deviate from [default settings](https://docs.djangoproject.com/en/4.2/ref/settings/#std-setting-STORAGES) in order to use `ManifestStaticFilesStorage`.

> The purpose of this storage is to keep serving the old files in case some pages still refer to those files, e.g. because they are cached by you or a 3rd party proxy server.
> — [Docker 4.2 @ `ManifestStaticFilesStorage`](https://docs.djangoproject.com/en/4.2/ref/contrib/staticfiles/#manifeststaticfilesstorage)